index.html

```diff
@@ -2,69 +2,120 @@
 <html>
 <head>
 	<title>HTML forms</title>
+	<link href="styles.css" rel="stylesheet" type="text/css">
+	<link href="https://fonts.googleapis.com/css?family=Nunito&display=swap" rel="stylesheet">
+	<meta charset="utf-8">
 </head>
 <body>
 	<main>
-		<div>
-			<div>
-				<div>
-					<div></div>
-					<a href=""></a>
+		<div class="sign-up" style="height: 60px;">
+			<div class="s-u-section">
+				<div class="s-u-left">
+					<div class="accnt-msg">Already have an Intuit account?</div>
+					<div class="accnt-msg">I forgot my User ID or Password</div>
 				</div>
-				<div>
-					<button>
-						<span>
+				<div class="s-u-right">
+					<button class="btn">
+						<span class="lock-img"></span> Sign In
 							
-						</span>
+						
 					</button>
 				</div>
 			</div>
 		</div>
 
-		<header>
-			<div>
-				<a href=""></a>
+		<header class="header-style">
+			<div class="logo-container">
+				<a href="" class="title-link">
+					
+				</a>
+				<img src="img/logoIntuit.png" class="logo-intuit">
 			</div>
 		</header>
 
 		<section>
 			<div>
 				<div>
 					<div class="icons">
-						<header>
-							<div>
+						<header class="header-style">
+							<div class="icons-container">
 								<a href=""></a>
-								<ul>
-									<li></li>
-									<li></li>
-									<li></li>
-									<li></li>
-									<li></li>
-									<li></li>
-									<li></li>
+								<ul class="form-icons">
+									<li><img src="img/intuit-mint-logo.png" style="padding: 1px;"></li>
+									<li class="icon-description">mint</li>
+									<li><img src="img/quickbooks-logo.png" style="padding-top: 1px;"></li>
+									<li class="icon-description">quickbooks</li>
+									<li><img src="img/turbotax-icon.png" style="padding-top: 1px;"></li>
+									<li class="icon-description">turbotax</li>
 								</ul>
 							</div>
 						</header>
 					</div>
-					<div>
-						<div></div>
-						<div></div>
-						<hr>
-						<section></section>
-					</div>
-					<footer>
-						<div>
-							<div>
+					<div class="form-area">
+						<section>
+							<header class="form-header">
+								<span class="create-acnt">Create An Intuit Account</span><br><br>
+								<span class="create-acnt-description">One account for everything Intuit,
+									 including Mint. </span>
+							</header>
+							<form>
+								<div>
+									<label class="form-label">Email address
```

Comment:
`AWESOME`

Good job nesting `<input>` element inside the `<label>` element. :clap: :+1: 

> #### Associating a `<label>` with an `<input>` element offers some major advantages:
> 
> - The label text is not only visually associated with its corresponding text input; it is programmatically associated with it too. This means that, for example, a screen reader will read out the label when the user is focused on the form input, making it easier for an assistive technology user to understand what data should be entered.
> 
> - You can click the associated label to focus/activate the input, as well as the input itself. This increased hit area provides an advantage to anyone trying to activate the input, including those using a touch-screen device.

An Alternative way to associate them is to give the `<input>` an id attribute. The `<label>` then needs a for attribute whose value is the same as the input's id. 
- Example : 

``` html
  <label for="email">Email address</label> <br>
  <input type="email" name="email" id="email">
```

You can learn more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)